src/hdf5/atms_netcdf_hdf5_2ioda.py

```diff
@@ -257,6 +260,28 @@ def get_WMO_satellite_ID(filename):
     return WMO_sat_ID
 
 
+def get_epoch_time(obs_time_utc):
+
+    # ugh this does not seem generic
+    year = obs_time_utc[:, :, 0].flatten()
+    month = obs_time_utc[:, :, 1].flatten()
+    day = obs_time_utc[:, :, 2].flatten()
+    hour = obs_time_utc[:, :, 3].flatten()
+    minute = obs_time_utc[:, :, 4].flatten()
+    second = obs_time_utc[:, :, 5].flatten()
+
+    # following examples here could be written better potentially
+    iterables = [year, month, day, hour, minute, second]
+    # ensure the year is plausible (65535 appears in some data) if not set to 01Jan1900 (revisit)
+    this_datetime = [datetime(adate[0], adate[1], adate[2], adate[3], adate[4], adate[5]) \
+        if adate[0] < 2200 else datetime(1900,1,1,0,0,0) \
```

Comment:
I'm not sure what is the correct value to be used as a `fillValue` for datetime, but I do see that people set it to `(2200, 1, 1, 0, 0, 0)` instead of the initial time (please, see for example https://github.com/JCSDA-internal/ioda/blob/develop/src/engines/Examples/Basic/07b-ObsSpaceClassDataTypes.py#L100).

```suggestion
        if adate[0] < 2200 else datetime(2200,1,1,0,0,0) \
```

I believe it would be good to add `fillValue` for datetime into the `VarAttrs` dictionary.